sdk/servicebus/service-bus/package.json

```diff
@@ -100,68 +88,95 @@
   },
   "dependencies": {
     "@azure/abort-controller": "^2.1.2",
-    "@azure/core-amqp": "^4.2.2",
-    "@azure/core-auth": "^1.3.0",
-    "@azure/core-client": "^1.7.0",
-    "@azure/core-paging": "^1.4.0",
-    "@azure/core-rest-pipeline": "^1.14.0",
-    "@azure/core-tracing": "^1.0.0",
-    "@azure/core-util": "^1.1.1",
-    "@azure/core-xml": "^1.0.0",
-    "@azure/logger": "^1.0.0",
+    "@azure/core-amqp": "^4.3.2",
+    "@azure/core-auth": "^1.8.0",
+    "@azure/core-client": "^1.9.2",
+    "@azure/core-paging": "^1.6.2",
+    "@azure/core-rest-pipeline": "^1.17.0",
+    "@azure/core-tracing": "^1.2.0",
+    "@azure/core-util": "^1.10.0",
+    "@azure/core-xml": "^1.4.3",
+    "@azure/logger": "^1.1.4",
     "@types/is-buffer": "^2.0.0",
     "buffer": "^6.0.0",
     "is-buffer": "^2.0.3",
-    "jssha": "^3.1.0",
+    "jssha": "^3.3.1",
     "long": "^5.2.0",
     "process": "^0.11.10",
     "rhea-promise": "^3.0.0",
-    "tslib": "^2.2.0"
+    "tslib": "^2.7.0"
   },
   "devDependencies": {
-    "@azure-tools/test-credential": "^1.0.0",
-    "@azure-tools/test-recorder": "^3.0.0",
-    "@azure-tools/test-utils": "^1.0.1",
+    "@azure-tools/test-credential": "^2.1.0",
+    "@azure-tools/test-recorder": "^4.1.0",
+    "@azure-tools/test-utils-vitest": "^1.0.0",
+    "@azure-tools/vite-plugin-browser-test-map": "^1.0.0",
     "@azure/dev-tool": "^1.0.0",
     "@azure/eslint-plugin-azure-sdk": "^3.0.0",
-    "@azure/identity": "^4.2.1",
+    "@azure/identity": "^4.4.1",
     "@microsoft/api-extractor": "^7.31.1",
-    "@types/chai": "^4.1.6",
-    "@types/chai-as-promised": "^7.1.0",
+    "@rollup/plugin-inject": "^5.0.5",
+    "@types/chai-as-promised": "^8.0.1",
     "@types/debug": "^4.1.4",
-    "@types/mocha": "^10.0.0",
     "@types/node": "^18.0.0",
-    "@types/sinon": "^17.0.0",
     "@types/uuid": "^8.0.0",
     "@types/ws": "^7.2.4",
-    "chai": "^4.2.0",
-    "chai-as-promised": "^7.1.1",
-    "chai-exclude": "^2.0.2",
+    "@vitest/browser": "^2.1.2",
+    "@vitest/coverage-istanbul": "^2.1.2",
+    "chai": "^5.1.1",
+    "chai-as-promised": "^8.0.0",
+    "chai-exclude": "^3.0.0",
     "cross-env": "^7.0.2",
     "debug": "^4.1.1",
     "dotenv": "^16.0.0",
     "eslint": "^9.9.0",
     "events": "^3.0.0",
     "https-proxy-agent": "^7.0.0",
-    "karma": "^6.2.0",
-    "karma-chrome-launcher": "^3.0.0",
-    "karma-coverage": "^2.0.0",
-    "karma-env-preprocessor": "^0.1.1",
-    "karma-firefox-launcher": "^1.1.0",
-    "karma-junit-reporter": "^2.0.1",
-    "karma-mocha": "^2.0.1",
-    "karma-mocha-reporter": "^2.2.5",
-    "karma-sourcemap-loader": "^0.3.8",
-    "mocha": "^10.0.0",
     "moment": "^2.24.0",
-    "nyc": "^17.0.0",
+    "playwright": "^1.46.1",
     "promise": "^8.0.3",
-    "puppeteer": "^23.0.2",
     "rimraf": "^5.0.5",
-    "sinon": "^17.0.0",
-    "ts-node": "^10.0.0",
+    "tshy": "^3.0.2",
```

Comment:
Does it have a dep?

Reply:
right, "tshy" is not needed any more with `dev-tool build-package`